pkg/resource/parameter_group/sdk.go

```diff
@@ -112,6 +118,15 @@ func (rm *resourceManager) sdkFind(
 	}
 
 	rm.setStatusDefaults(ko)
+	if ko.Status.ACKResourceMetadata != nil && ko.Status.ACKResourceMetadata.ARN != nil {
+		resourceARN := (*string)(ko.Status.ACKResourceMetadata.ARN)
+		tags, err := rm.getTags(ctx, *resourceARN)
+		if err != nil {
+			return nil, err
+		}
+		ko.Spec.Tags = tags
+	}
+
```

Comment:
There is no need to perform a nil check operation on `ko.Status.ACKResourceMetadata` and `ko.Status.ACKResourceMetadata.ARN` here. If the execution makes it to this block it means that a Parameter group exists and you'll have an ARN 100%. Check L116-L118